internal/tiltfile/files.go

```diff
@@ -188,7 +188,8 @@ func (s *tiltfileState) readFile(p localPath) ([]byte, error) {
 
 func (s *tiltfileState) skylarkReadFile(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var path starlark.Value
-	err := starlark.UnpackArgs(fn.Name(), args, kwargs, "path", &path)
+	defaultReturn := ""
+	err := starlark.UnpackArgs(fn.Name(), args, kwargs, "path", &path, "default?", &defaultReturn)
```

Comment:
While you're here, is it easy to also add it to the json function? (Totally okay to submit without doing that)

Reply:
Yeah I think I should just do that. Thanks for the reminder!